SalesforceSDK/Core/Rest/ApiVersionStrings.cs

```diff
@@ -29,7 +29,7 @@ namespace Salesforce.SDK.Rest
 {
     public class ApiVersionStrings
     {
-        public const string VersionNumber = "v31.0";
+        public const string VersionNumber = "v35.0";
```

Comment:
We could pull the api version from the bootconfig.json. That should probably be its own story: it would have to be done for all platforms and also from javascript ideally.